src/sidebar/services/load-annotations.js

```diff
@@ -180,7 +180,7 @@ export class LoadAnnotationsService {
       //    fetch the top-level annotation
       if (isReply(annotation)) {
         annotation = await this._api.annotation.get({
-          id: annotation.references[0],
+          id: /** @type {string[]} */ (annotation.references)[0],
```

Comment:
This is just asserting that `annotation.references` does exist here because we know the annotation is a reply.